pkgs/top-level/haskell-ng.nix

```diff
@@ -77,8 +81,9 @@ rec {
       packageSetConfig = callPackage ../development/haskell-modules/configuration-ghc-head.nix { };
     };
     ghcjs = callPackage ../development/haskell-modules {
+      ghc = compiler.ghcjs;
       packageSetConfig = callPackage ../development/haskell-modules/configuration-ghcjs.nix { };
+      jailbreak-cabal = "${packages.ghc7101.jailbreak-cabal}";
```

Comment:
I would prefer to override `jailbreak-cabal = pkgs.haskell-ng.packages.ghc7101.jailbreak-cabal` in `configuration-ghcjs.nix` instead.